src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -59,21 +59,29 @@ import ifndef from "../../shared/lib/ifndef";
 import {ISurvivalDescription} from "pages/resultsView/survival/SurvivalDescriptionTable";
 
 export enum OverlapStrategy {
-    INCLUDE = "Include overlapping samples and patients",
-    EXCLUDE = "Exclude overlapping samples and patients",
+    INCLUDE = "Include",
+    EXCLUDE = "Exclude",
 }
 
 export default class GroupComparisonStore {
 
     @observable private _currentTabId:GroupComparisonTab|undefined = undefined;
-    @observable private _overlapStrategy:OverlapStrategy = OverlapStrategy.EXCLUDE;
     @observable private sessionId:string;
     @observable public newSessionPending = false;
 
     constructor(sessionId:string, private appStore:AppStore, private routing:any) {
         this.sessionId = sessionId;
     }
 
+    @action public updateOverlapStrategy(strategy:OverlapStrategy) {
```

Comment:
is there a reason to do this in store?  if possible it seems to me the Page component is the right thing be updating router.

Reply:
but then the value is get and set in different places?

Reply:
Ill merge it for now because we're already doing a bunch of URL manipulation in the store anyway. So if we want to change that we can deal with it all at once.